addons/l10n_generic_coa/demo/account_invoice_demo.xml

```diff
@@ -82,6 +82,38 @@
         </record>
 
         <!-- ==== Post invoices ==== -->
+        <function model="account.move" name="_onchange_partner_id" context="{'check_move_validity': False}">
+            <value eval="[ref('demo_invoice_1')]"/>
+        </function>
+        <function model="account.move" name="_onchange_partner_id" context="{'check_move_validity': False}">
+            <value eval="[ref('demo_invoice_2')]"/>
+        </function>
+        <function model="account.move" name="_onchange_partner_id" context="{'check_move_validity': False}">
+            <value eval="[ref('demo_invoice_3')]"/>
+        </function>
+        <function model="account.move" name="_onchange_partner_id" context="{'check_move_validity': False}">
+            <value eval="[ref('demo_invoice_followup')]"/>
+        </function>
+        <function model="account.move" name="_onchange_partner_id" context="{'check_move_validity': False}">
+            <value eval="[ref('demo_invoice_equipment_purchase')]"/>
+        </function>
+
+        <function model="account.move.line" name="_onchange_product_id" context="{'check_move_validity': False}">
+            <value model="account.move.line" eval="obj().search([('move_id', 'in', [
+                ref('demo_invoice_1'), ref('demo_invoice_2'), ref('demo_invoice_3'),
+                ref('demo_invoice_followup'), ref('demo_invoice_equipment_purchase'),
+            ])]).ids"/>
+        </function>
+
+        <function model="account.move" name="_recompute_dynamic_lines" context="{'check_move_validity': False}">
+            <value eval="[
+                ref('demo_invoice_1'), ref('demo_invoice_2'), ref('demo_invoice_3'),
+                ref('demo_invoice_followup'), ref('demo_invoice_equipment_purchase'),
+            ]"/>
+            <value eval="True"/>
+            <value eval="True"/>
```

Comment:
?

Reply:
```python
def _recompute_dynamic_lines(self, recompute_all_taxes=False, recompute_tax_base_amount=False):
    ...
```
I'm just recomputing everything

Reply:
haha I realize I never seen/used a function call with more than 1 parameter...

can't you use 
```suggestion
            <value eval="True" name='recompute_all_taxes'/>
```
for more clarity? 

at least
```suggestion
            <value eval="True"/>    <!-- recompute_all_taxes parameter -->
```
should do the trick

Reply:
Weird, I'm almost certain I tried it before but the `name="..."` works indeed.
My brain is playing me